website/docs/d/lb_server_pool.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_lb_server_pool"
+sidebar_current: "docs-vcd-data-source-lb-server-pool"
+description: |-
+  Provides a NSX load balancer server pool data source.
+---
+
+# vcd\_lb\_server\_pool
+
+Provides a vCloud Director Edge Gateway Load Balancer Server Pool data source. A Server Pool defines
+a group of backend servers (defined as pool members), manages load balancer distribution methods, and has a service 
+monitor attached to it for health check parameters.
+
+~> **Note:** See additional support notes in [server pool resource page](/docs/providers/vcd/r/lb_server_pool.html).
+
+Supported in provider *v2.4+*
+
+## Example Usage
+
+```hcl
+data "vcd_lb_server_pool" "sp-ds" {
+  org                 = "my-org"
+  vdc                 = "my-org-vdc"
+  edge_gateway        = "my-edge-gw"
+
+  name = "not-managed"
+}
+```
```

Comment:
Just a reminder, that it would be cool to eventually see an extended sample how these data sources get actually used.

Reply:
So far I added it for lb_service_monitor (because now I can use it) in resource of lb_server_pool.
Also added a reference from datasource page.

lb_server_pool datasource can be used in next resource (I just need to remember :)